bundles/org.openhab.binding.luxtronikheatpump/src/main/resources/OH-INF/update/instructions.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes" ?>
```

Comment:
Great, now only the `thingTypeVersion` has to be set in thing-types.xml
```
  <properties>
    <property name="thingTypeVersion">1</property>
  </properties>
```

Refernece: https://www.openhab.org/docs/developer/bindings/thing-xml.html#updating-thing-types

Reply:
Done.